docs/_quarto.yml

```diff
@@ -187,118 +187,76 @@ quartodoc:
   sections:
     - title: Expression API
       desc: "APIs for manipulating table, column and scalar expressions"
-      package: ibis.expr.types
       contents:
         - kind: page
           package: ibis
-          path: top_level
+          path: connection
           summary:
-            name: Top-level APIs
-            desc: Methods and objects available directly on the `ibis` module.
+            name: Top-level backend API
+            desc: Create and manage backend connections.
           contents:
-            - name: and_
-              dynamic: true
-            - name: array
-              dynamic: true
-            - name: asc
-              dynamic: true
-            - name: case
-              dynamic: true
-            - name: coalesce
-              package: ibis.expr.api
             - name: connect
-              package: ibis.backends.base
-            - name: cumulative_window
-              dynamic: true
-            - name: date
-              dynamic: true
-            - name: desc
-              dynamic: true
-            - name: difference
-              dynamic: true
-            - name: dtype
               dynamic: true
             - name: get_backend
               dynamic: true
-            - name: greatest
-              package: ibis.expr.api
-            - name: ifelse
-              package: ibis.expr.api
-            - name: intersect
-              dynamic: true
-            - name: interval
-              dynamic: true
-            - name: least
-              package: ibis.expr.api
-            - name: literal
-              dynamic: true
-            - name: map
-              dynamic: true
-            - name: memtable
-              dynamic: true
-            - name: negate
-              package: ibis.expr.api
-            - name: NA
-              package: ibis.expr.api
-            - name: now
-              dynamic: true
-            - name: "null"
-              dynamic: true
-            - name: or_
-              dynamic: true
-            - name: param
-              dynamic: true
-            - name: show_sql
-              dynamic: true
-            - name: to_sql
-              dynamic: true
-            - name: random
-              dynamic: true
-            - name: range_window
+            - name: set_backend
```

Comment:
Actually can you add these in a follow up PR? It's hard to see the structural changes here without building the docs.

Reply:
Not quite sure what you mean by "these"? In my latest push I didn't move anything to underneath the Backends tab. Now just the "top-level" section only contains connection stuff and is renamed "Connection API". Hopefully this is clear enough.

I wish quarto had a preview feature for PRs like this, like readthedocs does... :(

Reply:
Oh, do you mean just remove commit b3429e9597a0ab73aaddf3a97e4db80345c5f768 from this PR and follow up with that later?

Reply:
Sorry! By "these" I mean the `set_backend`/`get_backend`/`connect` methods.

Reply:
I assumed so, so I removed it from the commit stack.

Reply:
Ah, now I see your clarification, thanks! I think now your concern is rendered useless, as now these methods aren't getting moved. Everything else is just getting moved out, and this section is getting renamed.